runtime/oti/VMHelpers.hpp

```diff
@@ -2070,57 +2089,87 @@ class VM_VMHelpers
 	static VMINLINE bool
 	isContinuationMounted(J9VMContinuation *continuation)
 	{
-		return J9_ARE_ANY_BITS_SET(continuation->state, ~(uintptr_t)J9_GC_CONTINUATION_STATE_CONCURRENT_SCAN);
+		return J9_ARE_ANY_BITS_SET(continuation->state, ~(uintptr_t)J9_GC_CONTINUATION_STATE_CONCURRENT_SCAN_ANY);
 	}
 
 	static VMINLINE bool
-	isContinuationMountedOrConcurrentlyScanned(J9VMContinuation *continuation)
+	isContinuationMountedOrConcurrentlyScanned(J9VMContinuation *continuation, bool isGlobalGC)
 	{
-		return isContinuationMounted(continuation) || isConcurrentlyScannedFromContinuationState(continuation->state);
+		return isContinuationMounted(continuation) || isConcurrentlyScannedFromContinuationState(continuation->state, isGlobalGC);
+	}
+
+	static VMINLINE uintptr_t getGCConcurrentState(bool isGlobalGC) {
```

Comment:
you could rename it to getGCConcurrentMask, but I'll leave it up to you

either way, you could use it at one more spot (where I got a motivation):
sConcurrentlyScannedFromContinuationState(uintptr_t continuationState, bool isGlobalGC)